src/system_info/system_info_display_wayland.cc

```diff
@@ -204,7 +204,7 @@ bool SysInfoDisplay::UpdateBrightness() {
   int val = atoi(str_val);
   free(str_val);
 
-  brightness_ = val / max_val;
```

Comment:
I'm asking what you had in mind. At first I thought this was a commit to fix the path name above, then I saw this change in the cast here that seems to fix something else related to the brightness code. It would be good to split this into two separate commits. And might even make more sense to just use `strtod()` instead of doing all those casts for this part.